week02/Largest Number.cpp

```diff
@@ -0,0 +1,26 @@
+// TC - o(nlogn)
+// MC - o(n)
+
+class Solution {
+public:
+    static bool compare(string a, string b) {
+        return a + b > b + a;
+    }
+
+    string largestNumber(vector<int>& nums) {
+
+        string Largest_Number = "";
+        vector<string> store;
+
+        for(auto c : nums)
+            store.push_back(to_string(c));
+
+        sort(store.begin(), store.end(), compare);

```

Comment:
Unnecessary. You could have done the string conversion and comparison inside the compare function.